commit 9155fdea19a0586fde11073cb1f1dd2f58d0b7b8
Author: Michael Terry <michael.terry@canonical.com>
Date:   Wed Mar 13 14:14:07 2013 +0000
Subject: Enable running tests during the build.


--- a/Makefile
+++ b/Makefile
@@ -65,7 +65,8 @@ uninstall:
 	;; esac
 
 tests/auto/Makefile: tests/auto/auto.pro
-	cd tests/auto && qmake -o Makefile auto.pro
+	cd tests/auto && qmake-qt4 -o Makefile auto.pro
+	cd tests/auto/qtchooser && qmake-qt4 -o Makefile qtchooser.pro
 
 check: tests/auto/Makefile
 	cd src/qtchooser && $(MAKE) check
